### DISCUSSION TITLE
http/Edge-Acceleration

### DISCUSSION THREAD

**Discussion by giscus** [isno/theByteBook#215] [General]
# http/Edge-Acceleration

构建大规模高可用的分布式系统

https://www.thebyte.com.cn/http/Edge-Acceleration.html

**Comment by echo1937** [comment]
这块上面感觉有些许惜字如金了，看了https://help.aliyun.com/zh/dcdn/product-overview/what-is-dcdn?spm=a2c4g.11186623.0.0.4cdf720bLiGMUy这部分才稍微了解大概的实现原理.

**Comment by PuPuuu** [comment]
“有关各类代理技术的原理，笔者将在本书第六章详细介绍” 貌似没有对这部分内容进行描述
